configure.ac

```diff
@@ -4098,8 +4098,10 @@ AS_VAR_IF([have_libffi], [yes], [
 # See https://github.com/python/cpython/issues/125206 for more details.
 #
 AC_CACHE_CHECK([libffi has complex type support], [ac_cv_ffi_complex_double_supported],
-[ac_save_cc="$CC"
-CC="$CC -lffi"
+[WITH_SAVE_ENV([
+ CFLAGS="$LIBFFI_CFLAGS $CFLAGS"
+ LDFLAGS="$LIBFFI_LIBS $LDFLAGS"
+ LIBS="$LIBFFI_LIBS"
```

Comment:
```suggestion
 LIBS="$LIBFFI_LIBS $LIBS"
```

Reply:
Hmm, which library we need else to run the test code?

Reply:
I would not expect any, but some build configurations are weird (cross-compile, etc.).